scripts/net-shaper.sh

```diff
@@ -15,7 +15,7 @@ fi
 
 set -x
 
-iface="$(ifconfig | grep mtu | grep -iv loopback | grep -i running | awk 'BEGIN { FS = ":" } ; {print $1}')"
```

Comment:
Should the `grep -i running` be replaced with `grep "state UP"` rather than omitted?

Reply:
Yes i think that's a better solution.

Reply:
Thanks!